bentoml/yatai/yatai_service.py

```diff
@@ -101,14 +101,15 @@ def start_yatai_service_grpc_server(
         ensure_node_available_or_raise()
         yatai_grpc_server_address = f'localhost:{grpc_port}'
         async_start_yatai_service_web_ui(
-            yatai_grpc_server_address, ui_port, web_ui_log_path, debug_mode
+            yatai_grpc_server_address, ui_port, web_ui_log_path, debug_mode, base_url
         )
 
     # We don't import _echo function from click_utils because of circular dep
     click.echo(
         f'* Starting BentoML YataiService gRPC Server\n'
         f'* Debug mode: { "on" if debug_mode else "off"}\n'
         f'* Web UI: {f"running on http://127.0.0.1:{ui_port}" if with_ui else "off"}\n'
+        f'* Web Prefix Path: {f"expecting to be hosted with prefix: {base_url}"}\n'
```

Comment:
I think this is a very confusing message to users.  If users didn't pass in a prefix path, the additional message on prefix is going to confuse them even more.

Let's just construct the web UI messages outside of the echo command and pass in.

Reply:
Let's make sure this is changed before we merge it.